tests/pebble-fetch.sh

```diff
@@ -13,15 +13,31 @@ mkdir -p ${PEBBLEPATH}
 
 cat << UNLIKELY_EOF > "$PEBBLEPATH/docker-compose.yml"
 version: '3'
-
 services:
- pebble:
-  image: letsencrypt/pebble:${PEBBLE_VERSION}
-  command: pebble -strict ${PEBBLE_STRICT:-false} -dnsserver 10.77.77.1
-  ports:
-    - 14000:14000
-  environment:
+  pebble:
+    image: letsencrypt/pebble
+    command: pebble -dnsserver 10.30.50.3:8053
+    environment:
+    - PEBBLE_VA_ALWAYS_VALID=1
```

Comment:
Why should we set this environment variable?

Reply:
Glitch from my tests, to be removed.

Reply:
Done